experiments/integrated/fluxnet/run_fluxnet.jl

```diff
@@ -87,8 +91,7 @@ soil_ps = Soil.EnergyHydrologyParameters(
     z_0m = z_0m_soil,
     z_0b = z_0b_soil,
     emissivity = soil_ϵ,
-    PAR_albedo = soil_α_PAR,
-    NIR_albedo = soil_α_NIR,
+    albedo = albedo,
```

Comment:
I think we should make the names a bit clearer in terms of what is soil_albedo vs. surface_albedo (canopy +soil), especially because we call emissivity = soil_e, so albedo = soil_albedo? Or is it surface_albedo?

Reply:
I will clarify this. In this case, it is in the soil parameter struct so it is the soil alone.

Reply:
I changed this also in ozark_pft